cdi/plugins/org.jboss.tools.cdi.gen/.project

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Plugin org.jboss.tools.cdi.gen is removed, probably Eclipse restored .project file. It should not be in this commit.